src/test/java/oshi/SystemInfoTest.java

```diff
@@ -240,6 +240,8 @@ public void testOSVersion() {
         assertNotNull(version.getVersion());
         assertNotNull(version.getCodeName());
         assertNotNull(version.getBuildNumber());
+        assertTrue(os.getOpenDescriptors() >= -1L);

```

Comment:
This is probably worthy of testing >=0 with a platform check for mac or linux.  See test for `getSystemLoadAverage()` for example.